setup.py

```diff
@@ -19,7 +31,7 @@
 
     setuptools.setup(
         name="queue_util",
-        version="2.2.1",
+        version=get_version(),
         author="Sujay Mansingh",
         author_email="sujay.mansingh@gmail.com",
```

Comment:
Might be worth changing the author and author email...

Reply:
43a1b69